src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -101,8 +105,47 @@ export default function applyAuthInterface(httpClient, authConfig) {
       // Attempt to refresh the JWT cookies.
       return httpClient.refreshAccessToken()
         // Successfully refreshed the JWT cookies, fire the callback function.
-        .then(() => {
-          callback(httpClient.getDecodedAccessToken());
+        .then((response) => {
+          const refreshedAccessToken = httpClient.getDecodedAccessToken();
+
+          /* istanbul ignore next */
+          if (refreshedAccessToken === null) {
```

Comment:
Note: we could test locally without this `if` just to ensure it all works as expected.  You might have done that.  Or, you might think that is overkill.  Anyway, just a thought.